broker/broker.go

```diff
@@ -252,6 +261,10 @@ func (b *CdnServiceBroker) parseProvisionDetails(details brokerapi.ProvisionDeta
 			return
 		}
 	}
+	if len(options.Headers) > 9 {
```

Comment:
Minor, but WDYT about checking header length after calling `getHeaders` instead, or inside the `getHeaders` method? We won't always forward the `Host` header, so sometimes users should be able to forward 10 headers instead of 9. It would also be useful to check the length of custom headers in one place (maybe in `getHeaders`) rather than two.

Reply:
I'd love to make this cleaner—it's been on my mind for months. I'll take a look!